core/core_integrations/slack.js

```diff
@@ -20,593 +20,593 @@ clearedTimeouts = false,
 };
 
 exports.start = function (callback) {
-	var slackTokens = exports.config.slack_tokens;
-
-	shuttingDown = false;
-	eventReceivedCallback = callback;
-	numSocketsToShutDown = 0;
-
-	console.debug("slack-> Starting slack output module");
-	if (slackTokens) {
-		for (var i = 0; i < slackTokens.length; i++) {
-			init(slackTokens[i], connect);
-		}
-
-		platform = shim.createPlatformModule({
-			sendMessage: sendMessage,
-			sendFile: sendFile,
-			setTitle: renameChannel,
-			sendTyping: sendTyping,
-			sendPrivateMessage: openPrivateMessage,
-			commandPrefix: exports.config.commandPrefix
-		});
-	}
-	else {
-		console.debug("slack-> Whoops looks like you don't have the slack config set up correctly");
-	}
+        var slackTokens = exports.config.slack_tokens;
+
+        shuttingDown = false;
+        eventReceivedCallback = callback;
+        numSocketsToShutDown = 0;
+
+        console.debug("slack-> Starting slack output module");
+        if (slackTokens) {
+                for (var i = 0; i < slackTokens.length; i++) {
+                        init(slackTokens[i], connect);
+                }
+
+                platform = shim.createPlatformModule({
+                        sendMessage: sendMessage,
+                        sendFile: sendFile,
+                        setTitle: renameChannel,
+                        sendTyping: sendTyping,
+                        sendPrivateMessage: openPrivateMessage,
+                        commandPrefix: exports.config.commandPrefix
+                });
+        }
+        else {
+                console.debug("slack-> Whoops looks like you don't have the slack config set up correctly");
+        }
 };
 
 exports.stop = function() {
-	console.debug("slack-> start shutdown");
-	shuttingDown = true;
-	numSocketsToShutDown = Object.keys(sockets).length;
-	clearedMessageQueue = false;
-	clearedTimeouts = false;
-
-	//Finish Message queue
-	// clear timeouts
-	// close sockets
-	async.series([
-		clearMessageQueue,
-		clearTimeouts,
-		closeSockets,
-		function(callback){
-			while (numSocketsToShutDown > 0 && !clearedMessageQueue && !clearedTimeouts) {
-				sleep.usleep(500000);
-			}
-			callback(null, "finished");
-		}
-	], function(err, results) {
-	});
+        console.debug("slack-> start shutdown");
+        shuttingDown = true;
+        numSocketsToShutDown = Object.keys(sockets).length;
+        clearedMessageQueue = false;
+        clearedTimeouts = false;
+
+        //Finish Message queue
+        // clear timeouts
+        // close sockets
+        async.series([
+                clearMessageQueue,
+                clearTimeouts,
+                closeSockets,
+                function(callback){
+                        while (numSocketsToShutDown > 0 && !clearedMessageQueue && !clearedTimeouts) {
+                                sleep.usleep(500000);
+                        }
+                        callback(null, "finished");
+                }
+        ], function(err, results) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['results' is defined but never used.](https://www.codacy.com/app/theprojectshop/Kassy/file/2555194709/issues/source?bid=3282987&fileBranchId=3291254#l610)